cbits/hschooks.c

```diff
@@ -34,7 +34,7 @@ defaultsHook (void)
     // This helps particularly with large compiles, but didn't work
     // very well with earlier GHCs because it caused large amounts of
     // fragmentation.  See rts/sm/BlockAlloc.c:allocLargeChunk().
-    RtsFlags.GcFlags.heapSizeSuggestionAuto = rtsTrue;
```

Comment:
Is this flag no longer used in 8.2.1? I could find almost no documentation on it.